multi_company_disable/static/src/js/backend.esm.js

```diff
@@ -1,8 +1,7 @@
 /** @odoo-module **/
-
-import { registry } from "@web/core/registry";
-import { } from "@web/webclient/switch_company_menu/switch_company_menu";
```

Comment:
I'm not sure why this import is here, but I removed it. If not ok, can you let me know why?

Reply:
Not sure either. Left over from development I guess.